libs/helpers.js

```diff
@@ -1,22 +1,26 @@
+const colors = require('colors');
 const opt = require('./../libs/models/options.js')
 
 function numberWithCommas(x) {
-    var parts = x.toString().split(".");
-    parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
-    return parts.join(".");
-  }
+  const parts = x.toString().split('.');
+  parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return parts.join('.');
```

Comment:
Didn't know which convention you preferred, single or double. Went with the most used one (singles)

Reply:
I'm fine with both

Reply:
Please remove the yarn.lock file for now

Reply:
I'll take care of it, don't worry